ScheduleSync/Professor.cs

```diff
@@ -20,7 +19,7 @@ public void ListarDisciplinasLecionadas()
             foreach (var disciplina in DisciplinasLecionadas)
             {
                 string professores = string.Join("/", disciplina.ProfessorResponsavel.Select(professor => professor.Nome));
-                Console.WriteLine(disciplina.Nome + " - " + professores);
+                Console.WriteLine($"{disciplina.Nome} - {professores}");
```

Comment:
_string interpolation_ for the win.

Reply:
Verdade, bem melhor, haha. Acabou passando, obrigado.